script.js

```diff
@@ -133,21 +133,21 @@ let deletePosts = async function (posts) {
         i++;
         console.log("Deleting post", post.date, post.author, "Completion:", (i*100/total) + "%");
         await deletePost(post.node);
-        await delay(1000);
+        await delay(3000);
     }
 };
 
 let scrollToBottom = async function (times) {
     // Let's scroll to the bottom a few times to load some content
     for (let i = 0; i < times; i++) {
         window.scrollTo(0,document.body.scrollHeight);
-        await delay(1000);
+        await delay(3000);
         console.log("Scrolling", (i*100/times) + "%")
     }
 };
 
 let main = async function () {
-    let HowManyRecentPostsToKeep = 6;
+    let HowManyRecentPostsToKeep = 0;
```

Comment:
I think it should be a positive integer as to not clear the whole group, usually people want the few latest ones to show.